scylla/src/transport/connection.rs

```diff
@@ -2406,7 +2406,7 @@ mod tests {
     use std::time::Duration;
 
     // Just like resolve_hostname in session.rs
-    async fn resolve_hostname(hostname: &str) -> SocketAddr {
+    pub(crate) async fn resolve_hostname(hostname: &str) -> SocketAddr {
         match tokio::net::lookup_host(hostname).await {
             Ok(mut addrs) => addrs.next().unwrap(),
```

Comment:
🔧 Instead of pub(crate)'ing it please move it to test_utils - this is what this module is for.